package/system/apk/Makefile

```diff
@@ -0,0 +1,66 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=apk
+PKG_RELEASE:=1
+
+PKG_SOURCE_DATE:=2024-03-16
+PKG_SOURCE_VERSION:=f2f10d7e851ed3ff2a239c89c1a6a3612adbd63f
+PKG_HASH:=964bbc71462630a5418a20ccd3e725eb0b065a6232d4aee16db745d95a04c1ff
+
+PKG_VERSION:=3.0.0_pre0
```

Comment:
We will need incremental preXXXXX versions because we will probably still bump to git HEAD a couple of times before the official APK 3.0.0 release.

Reply:
Can't we just use PKG_RELEASE for that?

Reply:
I added a patch so that the Git date is added to the binary itself